### PR TITLE
Fix reference to clip.geojson

### PR DESCRIPTION
**Pull request by madsbertelsen**

### CODE REVIEW

**Review by NyakudyaA — COMMENTED** [review]
Can you add a missing backtick so that the complete section becomes  code block

File: readme.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ As a quick example, we are going to setup Docker-OSM with default values everywh
 * Put the file in the `settings` folder and rename it `country.pbf`. 
 
 Alternatively you can execute the `settings_downloader.sh` script to download the pbf and the clip file
-``
+```
 ./settings_downloader.sh GEOJSON_URL CONTINENT COUNTRY ie
 ./settings_downloader.sh https://github.com/kartoza/docker-osm/raw/develop/settings/clip.geojson africa south-africa
 ``
```
Comment (Gustry) [inline readme.md:22]:
```suggestion
```
```

Comment (Gustry) [inline readme.md:22]: if you add one on the top, you need one L22

Thanks for that.

**Review by Gustry — APPROVED** [review]

File: readme.md (hunk 1 above)
Comment (madsbertelsen) [reply readme.md]: Sure, no problem. Should be good now.

**Pull request merged**